openshift_performance/ose3_perf/scripts/build_test.py

```diff
@@ -89,6 +89,7 @@ def parse(executor, result, futures):
             status = words[4]
             namespace = words[0]
             name = words[1]
+            duration = words[len(words)-1]
```

Comment:
Could we move `duration_string = words[-1]` in line 107 up here?

Reply:
@hongkailiu Good catch, I completely missed that.   Commit updated.